README.md

```diff
@@ -1,121 +1,100 @@
-
-# Shabad OS Database Viewer
-
-<div align="center">
-  <img src="./repologo.png">
-</div>
-<br/>
-<div align="center">
-
-Work in-progress: An online viewer of the bani database. Authenticated by GitHub for suggesting changes. Shown as pull requests to our [Database](https://github.com/ShabadOS/database) repo.
```

Comment:
`Authenticated by GitHub for suggesting changes. Shown as pull requests to our [Database](https://github.com/ShabadOS/database) repo.` Some of this blurb i thought was actually quite useful.

Reply:
It's not authenticated in our website, it's just a link to open an issue

They are not shown as pull requests to our database repo

None of this was implemented

Reply:
That's fair enough - I think saying `opens issues in [link]` just so that readers understand the relationship between this and DB

Reply:
I think the goal with the viewer is to decouple it from being a proofreading tool only, to being more of it's own entity the way presenter is

The presenter also allows one to submit a proofreading ticket. So it's not unique to the viewer, but I've still added this to the "features" section, idk if you noticed:

> Submit issue tickets for any line while proofreading

I don't think the reader of a readme cares where the ticket goes

Reply:
I'm ok with it being in features in that case. I'd still link to database issues, one may wish to have a quick browse out of piqued interest, no?

Reply:
Please [see updated readme](https://github.com/ShabadOS/viewer/blob/8eac917040145a7b299247787a3bfc2dfb27ccf7/README.md#features)